test/runtests.jl

```diff
@@ -72,13 +72,15 @@ end
         # Tests spatial modelling and simulations.
         @time @safetestset "PDE Systems Simulations" begin include("spatial_modelling/simulate_PDEs.jl") end
         @time @safetestset "Spatial Reactions" begin include("spatial_modelling/spatial_reactions.jl") end
-
-        # BROKEN
-        #@time @safetestset "Lattice Reaction Systems" begin include("spatial_modelling/lattice_reaction_systems.jl") end
+        @time @safetestset "Lattice Reaction Systems" begin include("spatial_modelling/lattice_reaction_systems.jl") end
         @time @safetestset "Spatial Lattice Variants" begin include("spatial_modelling/lattice_reaction_systems_lattice_types.jl") end
         @time @safetestset "ODE Lattice Systems Simulations" begin include("spatial_modelling/lattice_reaction_systems_ODEs.jl") end
         @time @safetestset "Jump Lattice Systems Simulations" begin include("spatial_modelling/lattice_reaction_systems_jumps.jl") end
         @time @safetestset "Lattice Simulation Structure Interfacing" begin include("spatial_modelling/lattice_simulation_struct_interfacing.jl") end
+
+        # Test spatial plotting, using CairoMakie and GraphMakie
+        activate_extensions_env()
+        @time @safetestset "Lattice Simulation Plotting" begin include("extensions/lattice_simulation_plotting.jl") end
```

Comment:
Figured while this was placed in an extension (to not require Makie stuff generally) it really is a part of the spatial stuff, so might make sense to move here?
(but I am also happy to have it in the previous place)

Reply:
I'd prefer to keep anything loading extensions in one place given the high frequency at which they seem to give failures, and to avoid blocking our other tests via extension related dependency issues.

Reply:
sounds good, will move it back then